### ISSUE TITLE
AddSliceContainer.jsx can't be translated by messages.json 

### ISSUE THREAD

**Issue by cdmikechen** [apache/incubator-superset#8749]
### Expected results
AddSliceContainer.jsx can be translated  

### Actual results
All other pages can be translated correctly, except `AddSliceContainer.jsx` cannot.

#### Screenshots
No.

#### How to reproduce the bug
I run a development flask and dev-server in my computer to test translation. I found that `AddSliceContainer.jsx` can not be translated well, meanwhile other pages can be translated correctly.

### Environment

- superset version: `0.35` and `0.34`
- python version: `3.7`
- node.js version: `v10.16.0`
- npm version: `6.9.0`

### Checklist

Make sure these boxes are checked before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

### Additional context
No.

**Comment by cdmikechen** [comment]
Finally found the cause of the problem~
1. In `superset/assets/src/preamble.js` superset must get document id `app` to set `@superset-ui/translation` config. But  `AddSliceContainer.jsx` replace id `app` to `js-add-slice-container`.
2. In `superset/views/core.py` web response must contain `bootstrap_data.common` node, so that `superset/assets/src/preamble.js` can read translation messages.
```
"common": self.common_bootstrap_payload(),
```
But `/chart/add` response data doesn't have this node.
```python
        return self.render_template(
            "superset/add_slice.html",
            bootstrap_data=json.dumps(
                {"datasources": sorted(datasources, key=lambda d: d["label"])
            ),
        )
```
We need to add this element like this:
```python
        messages = get_flashed_messages(with_categories=True)
        locale = str(get_locale())
        common_bootstrap = {
            "flash_messages": messages,
            "conf": {k: conf.get(k) for k in FRONTEND_CONF_KEYS},
            "locale": locale,
            "language_pack": get_language_pack(locale),
            "feature_flags": get_feature_flags(),
        }

        return self.render_template(
            "superset/add_slice.html",
            bootstrap_data=json.dumps(
                {"datasources": sorted(datasources, key=lambda d: d["label"]),
                 "common": common_bootstrap}
            ),
        )
```

**Comment by mistercrunch** [comment]
Good catch. Please send a PR. I think you could just:
```python
        return self.render_template(
            "superset/add_slice.html",
            bootstrap_data=json.dumps(
                {
                    "datasources": sorted(datasources, key=lambda d: d["label"]),
                    "common": common_bootstrap_payload(),
                }
            ),
        )
```

At some point we'll be starting to move more towards a single page app, but in the meantime we should fix this.

**Comment by auxten** [comment]
I just fixed this issue again #9321

**Comment by chengkaisheng** [comment]
It helps. Thanks!
